README.md

```diff
@@ -127,6 +127,17 @@ See http://semver.org/.
 
 Please check [CONTRIBUTING](CONTRIBUTING.md) before making a contribution.
 
+## Release flow
+
+As of 2019-12, releaser is LINE's employee.
```

Comment:
I'm pretty much not sure this line is needed or not.

Reply:
Exactly. Let me delete it.